test/testdata/runner/router_tests.test.json

```diff
@@ -13,11 +13,11 @@
                 {
                     "category": "Other",
                     "created_on": "2018-07-06T12:30:11.123456789Z",
-                    "input": "Ben Haggerty",
+                    "input": "[]",
```

Comment:
result input is the operand stringified.. need to think about implications of removing operand on switch routers here